HomeWork18/HomeWork18/Models/Employee.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace HomeWork18.Models
+{
+    internal class Employee : Validation
+    {
+        public string Name { get; set; } = null!;
```

Comment:
What does this mean? null!

Reply:
null! operator git work aplication with out Exception and other diferent error